tcpb/utils.py

```diff
@@ -121,6 +169,8 @@ def job_output_to_atomic_result(
     atomic_result.extras.update(
         {
             "qcvars": {
+                "energy": jo_dict.get("energy"),
```

Comment:
This should likely be removed as per the note above re: return energy being on `result.properties.return_energy`